scalding-core/src/main/scala/com/twitter/scalding/Operations.scala

```diff
@@ -476,7 +476,21 @@ package com.twitter.scalding {
     }
   }
 
-  /** In the typed API every reduce operation is handled by this Buffer */
+  private[scalding] object SkewMonitorCounters {
+    // Strangely, if group name and key name are different, then the counter would be zero
```

Comment:
This seems to be a bug. if you simply put 

flowProcess.increment("TestGroup", "TestKey", 1)

in     def operate(flowProcess: FlowProcess[_], call: BufferCall[Any]) {

and this should print 3 in the test

ReduceValueCounterTest. However it prints

PRINTING KEY AND GROUP! 0